assets/js/components/TourTooltips.test.js

```diff
@@ -452,4 +452,29 @@ describe( 'TourTooltips', () => {
 			);
 		} );
 	} );
+
+	it( 'should set stepIndex to 0 when tour is started and step key is undefined', async () => {
+		registry.dispatch( CORE_UI ).setValue( STEP_KEY, undefined );
+
+		const { getByRole } = renderTourTooltipsWithMockUI( registry );
+		await getByRole( 'alertdialog' );
+		mockTrackEvent.mockClear();
+
+		// Go to step 2
+		fireEvent.click( getByRole( 'button', { name: /next/i } ) );
+		await getByRole( 'alertdialog' );
+		// Tracks the advance on the 1st step, view on the 2nd step.
+		expect( mockTrackEvent ).toHaveBeenNthCalledWith(
+			1,
+			EVENT_CATEGORY,
+			GA_ACTIONS.NEXT,
+			1
+		);
+		expect( mockTrackEvent ).toHaveBeenNthCalledWith(
+			2,
+			EVENT_CATEGORY,
+			GA_ACTIONS.VIEW,
+			2
+		);
+	} );
```

Comment:
I feel the test case here doesn't sufficiently verify the change here. This is also placed within the `event tracking` section, which is not correct.

Instead, I think we should pass a mock callback function to the `TourTooltips` `callback` prop and verify that it was called with the appropriate `data`, i.e. `controlled` is `true`, and/or `step` is `0`.